designs/add-recap-search-option-about-representation/readme.md

```diff
@@ -6,82 +6,94 @@
 - Problem statement
 - Proposed solution by component
   - User Interface
-  - Django code for parsing and storing docket data
+  - Django code for analyzing and storing docket data
   - Solr
   - Already-stored docket data
   - Documentation
 - Test cases
 
 # Problem statement
 
-PACER and RECAP search users do not currently have an option to search for dockets where at least one petitioner had representation. Such dockets generally have higher quality complaints, motions, briefs, citations, and prose. Because that option is not available, searchers have to use other means that cost time and money to differentiate such cases from those where every petitioner is pro se.
+PACER and RECAP search users do not currently have an option to search for dockets where at least one petitioner had representation. Searchers like to examine those dockets because they generally have higher quality complaints, motions, briefs, citations, and prose. Because that option is not available, searchers have to use other means that cost time and money to differentiate such cases from those where every petitioner is pro se.
 
 # Proposed solution 
 
-Enable a RECAP search option for retrieving dockets where at least one petitioner had representation.
+Enable a RECAP search option to exclude cases from the results where every petitioner was pro se.
 
-Enabling the option requires changes in (1) the user interface, (2) the server-side Django code that parses and stores docket data, (3) the Solr search configuration, (4) the already-stored docket data, and (5) documentation.
+Enabling the option requires changes in (1) the user interface, (2) the server-side Django code that analyzes and stores docket data, (3) the Solr search configuration, (4) the already-stored docket data, and (5) documentation.
 
-The RECAP browser extension will not need to be changed.
+Neither the RECAP browser extension nor Juriscraper will need to be changed.
 
 ## Proposed solution: User Interface
 
-On the CourtListener.com [RECAP search page](https://www.courtlistener.com/recap/), **add** a select list to the search options.  The list's label will be: "At least one petitioner is represented".
+On the CourtListener.com [RECAP search page](https://www.courtlistener.com/recap/), **add** a checkbox to the search options.  The checkbox's label will be: "Exclude *pro se* petitioner cases".
 
-**Provide** three select list options.  The first empty/blank value will be the default.
-- ""
-- "Yes"
-- "No (Every party is pro se)"
+Also, **add** a pop-up help tip through a question-mark icon with this text:
+"If this box is checked, then cases where every petitioner is pro se will be excluded from the search results.  Parties treated as petitioners include plaintiffs, appellants, applicants, complainants, claimants, and relators. If a party's name does not match the corresponding attorney's name or no attorney name is provided, then the party is assumed to be pro se."
```

Comment:
That's great on a technical level, but most people don't care about such details. What they do care about is, "Why should I check this box?" And that's still a gap that we need to address. Maybe lead the explainer with something like:

> Pro se petitioner cases are often of a lower quality than those with professional representation. If this box is checked, then cases where...

Or something to that effect.